core/src/main/java/io/questdb/griffin/engine/functions/groupby/CountDistinctUuidGroupByFunction.java

```diff
@@ -74,12 +71,13 @@ public void computeFirst(MapValue mapValue, Record record) {
             mapValue.putLong(valueIndex + 1, setA.ptr());
         } else {
             mapValue.putLong(valueIndex, 0);
-            mapValue.putLong(valueIndex + 1, 0);;
+            mapValue.putLong(valueIndex + 1, 0);
+            ;
```

Comment:
nit, suprlus character

Reply:
Removed in bdabd26b047ddb61a766eae93166770d53e564f8